src/components/PostItem.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Post } from '../types/Post';
+
+type Props = {
+  post: Post,
+  isSelected: boolean,
+  setSelectedPost: (post: Post | null) => void,
+};
+
+export const PostItem: FC<Props> = ({ post, isSelected, setSelectedPost }) => {
+  const handlePostButton = () => {
+    setSelectedPost(isSelected ? null : post);
+  };
+
+  return (
+    <tr key={post.id} data-cy="Post">
```

Comment:
Remove the 'key' prop from the 'tr' element, as it is not needed in this component. It should be used when mapping over an array of elements in the parent component.